lib/Doctrine/ORM/EntityManager.php

```diff
@@ -248,6 +248,28 @@ public function transactional($func)
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function wrapInTransaction(callable $func)
```

Comment:
Shouldn't `transactional()` be deprecated in favor of this method?

Reply:
Aight, tried to deprecate it :)